### PR TITLE
feat: Add URLs for custom events and URL filtering.

### PR DESCRIPTION
**Pull request by kinyoklion**

### CODE REVIEW

File: packages/sdk/browser/src/goals/LocationWatcher.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const LOCATION_WATCHER_INTERVAL = 300;
```
Comment (kinyoklion) [inline packages/sdk/browser/src/goals/LocationWatcher.ts:1]: Meant to address this in the other PR, but catching it here.

File: packages/sdk/browser/src/index.ts [hunk 2]
```diff
@@ -36,10 +34,7 @@ export {
...
 
-export function init(
-  clientSideId: string,
-  autoEnvAttributes: AutoEnvAttributes,
-  options?: LDOptions,
-): LDClient {
-  return new BrowserClient(clientSideId, autoEnvAttributes, options);
+export function init(clientSideId: string, options?: LDOptions): LDClient {
+  // AutoEnvAttributes are not supported yet in the browser SDK.
+  return new BrowserClient(clientSideId, AutoEnvAttributes.Disabled, options);
```
Comment (kinyoklion) [inline packages/sdk/browser/src/index.ts:39]: When we add browser auto attributes we add this.

File: packages/sdk/browser/src/platform/BrowserInfo.ts [hunk 3]
```diff
@@ -10,8 +8,8 @@ export default class BrowserInfo implements Info {
   }
   sdkData(): SdkData {
     return {
-      name,
-      version,
+      name: '@launchdarkly/js-client-sdk',
```
Comment (kinyoklion) [inline packages/sdk/browser/src/platform/BrowserInfo.ts:11]: This is to make builds simpler.

File: packages/sdk/browser/src/platform/BrowserPlatform.ts [hunk 4]
```diff
@@ -22,9 +22,9 @@ export default class BrowserPlatform implements Platform {
   requests: Requests = new BrowserRequests();
   storage?: Storage;
 
-  constructor(options: LDOptions) {
+  constructor(logger: LDLogger) {
```
Comment (kinyoklion) [inline packages/sdk/browser/src/platform/BrowserPlatform.ts:25]: Decrease scope of dependency.

**Review by tanderson-ld — APPROVED** [review]

**Pull request merged**